src/URLS.tsx

```diff
@@ -51,7 +51,7 @@ const URLS = {
   profile: '/profil/',
   signup: '/ny-bruker/',
   shortLinks: '/linker/',
-
+  gallery: '/albumer/',
```

Comment:
```suggestion
  gallery: '/album/',
```

Flertall av album er også album
*Ett album, flere album*

Kan også vurdere å heller bruke `/galleri/` som url, hva tenker du?